src/sql/plugins/mylutece/modules/users/plugin/create_db_mylutece-users.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- Structure for table mylutece_users_localuser
+--
+DROP TABLE IF EXISTS mylutece_users_localuser;
+CREATE TABLE mylutece_users_localuser (
+id_local_user int AUTO_INCREMENT,
+login varchar(255) default '' NOT NULL,
```

Comment:
"connect_id" or "user_name" should be more coherent with Lutece history

Reply:
ok :/